user_guide_src/source/models/model.rst

```diff
@@ -187,7 +187,7 @@ data type.
 
 **$createdField**
 
-Specifies which database field should use for keep data record create timestamp.
+Specifies which database field to use for data record create timestamp.
```

Comment:
"data record create timestamp" seems off to me.

Reply:
Yes, I agree with you. Just didn't want to go farther before getting positive feedback.

Reply:
Ok then. Can you please go further and edit "data record create timestamp" to make it less confusing?